protocol/communities/persistence.go

```diff
@@ -1401,3 +1413,36 @@ func toRevealedAccount(rawAddress sql.NullString, rawChainIDsStr sql.NullString,
 	}
 	return revealedAccount, nil
 }
+
+func (p *Persistence) GetSyncControlNode(communityID types.HexBytes) (*protobuf.SyncCommunityControlNode, error) {
+	result := &protobuf.SyncCommunityControlNode{}
+
+	err := p.db.QueryRow(`
+        SELECT clock, installation_id
+        FROM communities_control_node
+        WHERE community_id = ?
+    `, communityID).Scan(&result.Clock, &result.InstallationId)
+
+	if err != nil {
+		if err == sql.ErrNoRows {
+			return nil, nil
+		}
+		return nil, err
+	}
+
+	return result, nil
+}
+
+func (p *Persistence) SaveSyncControlNode(communityID types.HexBytes, clock uint64, installationID string) error {
+	_, err := p.db.Exec(
+		`INSERT INTO communities_control_node (
```

Comment:
Is this new table necessary? Could we just have added the installation_id in the communities table (with an update clock if needed)

Reply:
Yes. The clock for community and for sync messages is different (and should be different).

Reply:
Yeah I get it. We can add another field called `installation_id_updated_at` though if we want.

Reply:
Right, it would work as well. I didn't want to pollute communities table with something that is relevant only to control nodes.

Reply:
I don't have a strong opinion. I think adding a new table can get a little more confusing, but it's also a bit cleaner like you said since it's only for owners. I'll let the other reviewer decide :stuck_out_tongue: